netutils/config/parser.py

```diff
@@ -1375,3 +1375,43 @@ def build_config_relationship(self) -> t.List[ConfigLine]:
 
             self._update_config_lines(line)
         return self.config_lines
+
+
+class OptiswitchConfigParser(BaseSpaceConfigParser):
+    """MRV Optiswitch config parser."""
+
+    comment_chars: t.List[str] = ["#", "!"]
+    banner_start: t.List[str] = []
+
+    def __init__(self, config: str):
+        """Create ConfigParser Object.
+
+        Args:
+            config (str): The config text to parse.
+        """
+        super(OptiswitchConfigParser, self).__init__(config)
```

Comment:
I see that we did this on Nokia and Aruba, I think it was just missed. I don't think this is needed.

Reply:
Thanks, removed both of these sections 👍